src/utils/proxies/__init__.py

```diff
@@ -62,15 +62,15 @@
         "name": "Cisco Talos",
         "url": "https://talosintelligence.com/reputation_center",
         "categorize_url": "https://talosintelligence.com/reputation_center",
-        "check_url": "https://talosintelligence.com/reputation_center",
+        "check_url": None,
         "check_category_func": None,
         "categorize_func": None,
     },
     {
         "name": "IBM X Force",
         "url": "https://exchange.xforce.ibmcloud.com/",
         "categorize_url": "https://exchange.xforce.ibmcloud.com/",
-        "check_url": "https://exchange.xforce.ibmcloud.com/",
```

Comment:
Why did you remove the check_urls in these two locations? The check url is only used for the frontend so that somebody can go manually check it.

Reply:
I didn't know it was used for that. It was failing in the proxy test harness because its checking if that variable exists before it runs the category check.

Reply:
Ahh, yes, it would fail as I'm looking at it. Let me check how I fixed it when we actually process it.

Reply:
Actually should be good if it's added back in. It looks like the `process` function will fail, but on exception it returns None instead of the category. Which is what should happen.